src/drivers/imu/adis16477/adis16477_main.cpp

```diff
@@ -31,123 +31,114 @@
  *
  ****************************************************************************/
 
-#include "ADIS16477.hpp"
+#include "ADIS16477.h"
```

Comment:
@dagar, this needs to be restored to `*.hpp` or the file extension changed.

Nice cleanup!

Reply:
Thanks Mark, fixed.